Makefile

```diff
@@ -0,0 +1,40 @@
+NAME ?= "KialiTestService"
```

Comment:
test service or should it be test mesh like the repo name?

Reply:
service. A mesh is composed of multiple services, and this just builds one service.

I wasn't the one who voted to call it 'kiali-test-mesh' :)

Reply:
I think it was more of a question that got answered so I went ahead and merged. If I am wrong we can rename in a separate PR.